ogr/services/github/pull_request.py

```diff
@@ -123,14 +123,18 @@ def create(
         source_branch: str,
         fork_username: str = None,
     ) -> "PullRequest":
+        """        
+        The default behaviour is pull request is made to the immediate parent repository
+        if the repository is a forked repository.
+        If you wanto create an pull request to the forked repo, please pass 
+        the `fork_username` parameter. 
+        """
         github_repo = project.github_repo
 
-        if project.is_fork and fork_username:
-            logger.warning(
-                f"{project.full_repo_name} is fork, ignoring fork_username arg"
-            )
+        if project.is_fork and fork_username is None:
+            logger.warning(f"{project.full_repo_name} is fork, ignoring fork_repo.")
```

Comment:
Since you have the same condition below, could you please move the warning there? :) Thanks

Reply:
Thanks done :)